rules/scripts.mk

```diff
@@ -23,11 +23,15 @@ $(SYSCTL_NET_CONFIG)_PATH = files/image_config/sysctl
 UPDATE_CHASSISDB_CONFIG_SCRIPT = update_chassisdb_config
 $(UPDATE_CHASSISDB_CONFIG_SCRIPT)_PATH = files/scripts
 
+SWSS_VARS_TEMPLATE = swss_vars.j2
+$(SWSS_VARS_TEMPLATE)_PATH = files/build_templates
+
 SONIC_COPY_FILES += $(CONFIGDB_LOAD_SCRIPT) \
```

Comment:
>SONIC_COPY_FILES [](start = 0, length = 16)

Why we don't need to change this before?

Reply:
The template file was previously put in the docker-orchagent directory, so it could be seen by orchagent docker. Since we want to put this file into more than one docker, I moved it into a shared place and added this change.